src/components/five-day-weather.jsx

```diff
@@ -0,0 +1,40 @@
+import { getWeatherPic } from "../images/index";
+import React from "react";
+
```

Comment:
I notice you're using a lot of class components and lifecycle methods. How familiar are you with using hooks to create stateful functional components?

Reply:
I am somewhat familiar with React hooks, but I mostly work with Angular and so that’s where a lot of my current expertise lies. However I did do my bootcamp with React and I like working on side projects using it, but my mind was definitely in that component-first mode that comes with Angular. Next time I would convert some of the smaller components, like the search bar, to hooks.